db/sql/migrations/updates/20180205000003_create_rounds_rewards_table.sql

```diff
@@ -38,6 +38,9 @@ CREATE TABLE "rounds_rewards"(
 	"publicKey" BYTEA  NOT NULL
 );
 
+-- Insert record for dummy round 1 when genesis block is in database
+INSERT INTO rounds_rewards SELECT timestamp, 0, 0, 1, "generatorPublicKey" FROM blocks WHERE height = 1;
+
```

Comment:
Please change the query to 

```sql
INSERT INTO rounds_rewards (timestamp, fees, reward, round, "publicKey") 
      SELECT timestamp, 0, 0, 1, "generatorPublicKey" FROM blocks WHERE height = 1;
```
That makes more readable to know which values goes to which columns.

Reply:
Table structure is defined just 1 line above, but ok - I'll change that.